README.md

```diff
@@ -19,15 +19,15 @@ My PLDI'18 paper includes a complete correctness proof of the algorithm:
 https://dl.acm.org/citation.cfm?id=3192369, available under the creative commons
 CC-BY-SA license.
 
-Andriy Plokhotnyuk developed a scala implementation that is available here:
-https://github.com/plokhotnyuk/jsoniter-scala
-
-Mara Bos provides a Rust wrapper of the C code here:
-https://docs.rs/ryu/0.1.1/ryu/
-
-Julia version: https://github.com/quinnj/Ryu.jl
-
-Factor version: https://github.com/AlexIljin/ryu
+Other implementations:
+
+| Language         | Author             | Link                                          |
+|------------------|--------------------|-----------------------------------------------|
+| Scala            | Andriy Plokhotnyuk | https://github.com/plokhotnyuk/jsoniter-scala |
+| Rust (C wrapper) | Mara Bos           | https://docs.rs/ryu/0.1.1/ryu/                |
```

Comment:
The Rust library was transferred from Mara to myself (see https://github.com/m-ou-se/ryu-rust/blob/master/README) and replaced with a pure Rust port that lives at https://github.com/dtolnay/ryu.

Reply:
I merged this and then updated the link as you described. Thanks!